src/dreyfus/src/dreyfus_httpd.erl

```diff
@@ -209,6 +212,16 @@ parse_index_params(IndexParams, Db) ->
         validate_index_query(K, V, Args2)
     end, Args, IndexParams).
 
+unwrap_list_of_list_of_lists([Inner]=Outer) when is_list(Inner) ->
```

Comment:
Can you rename this to what its actually doing and leave a comment explaining the list of lists of lists thing with drilldowns.